src/community/features-templating/features-templating-core/src/test/java/org/geoserver/featurestemplating/writers/JSONLDWriterTest.java

```diff
@@ -0,0 +1,87 @@
+package org.geoserver.featurestemplating.writers;
+
+import static org.geoserver.featurestemplating.builders.VendorOptions.COLLECTION_NAME;
+import static org.geoserver.featurestemplating.builders.VendorOptions.JSONLD_TYPE;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.fasterxml.jackson.core.JsonEncoding;
+import com.fasterxml.jackson.core.JsonFactory;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import net.sf.json.JSONObject;
+import net.sf.json.JSONSerializer;
+import org.geoserver.featurestemplating.builders.EncodingHints;
+import org.junit.Test;
+
+public class JSONLDWriterTest {
+
+    @Test
+    public void testJSONRootAttributesTypeCustomization() throws IOException {
```

Comment:
Confused by the test name, this one seems to customize the collection name, not the type?

Reply:
fixed